File: spl/10-print-the-sum-of-following-series/solution.c
#include<stdio.h>;

int main()
{
    int i, n, sum = 0;

    printf("Enter the n i.e. max values of series: ");
    scanf("%d",&n);

    for(i=1; i<=n; i *= 3){
        sum += i;
        printf("%d \n", i);
    }

    printf("Sum of the series: %d", sum);
    return 0;
}

File: spl/10-print-the-sum-of-following-series/solution.php
<?php
    $n=readline("Enter the max values of the series : \n");
    $sum=0;
    for($i=1; $i<$n;$i*=3){
        $sum=$sum+$i;
        echo $i."\n";
    }
    echo "Sum of the series 1+3+9+....".$n. " is = ". $sum."\n";
?>